pyproject.toml

```diff
@@ -1,6 +1,39 @@
 [build-system]
-requires = ['setuptools >= 40.8.0', 'wheel']
-build-backend = 'setuptools.build_meta'
+requires = ["flit_core >=3.3"]
+build-backend = "flit_core.buildapi"
+
+
+[project]
+name = "packaging"
+description = "Core utilities for Python packages"
+version = "21.4.dev0"
+readme = "README.rst"
+requires-python = ">=3.7"
+authors = [{name = "Donald Stufft", email = "donald@stufft.io"}]
+classifiers = [
+  "Development Status :: 5 - Production/Stable",
+  "Intended Audience :: Developers",
+  "License :: OSI Approved :: Apache Software License",
+  "License :: OSI Approved :: BSD License",
+  "Programming Language :: Python",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3.7",
+  "Programming Language :: Python :: 3.8",
+  "Programming Language :: Python :: 3.9",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: Implementation :: CPython",
+  "Programming Language :: Python :: Implementation :: PyPy",
+]
+urls.Source = "https://github.com/pypa/packaging"
+dependencies = [
+  "pyparsing>=2.0.2,!=3.0.5",
+]
+
+
+[tool.flit.sdist]
+include = ["LICENSE*", "tests/", "docs/"]
+exclude = ["docs/_build", "tests/manylinux/build-hello-world.sh", "tests/musllinux/build.sh", "tests/hello-world.c", "tests/__pycache__", "build/__pycache__"]
```

Comment:
I don't think we need to exclude `__pycache__`.

Reply:
`__pycache__` is only automatically excluded from the package folder.

Reply:
We might want to report another issue about that... I don't think it makes any sense for sdist to contain Python bytecode.

Reply:
> I don't think it makes any sense for sdist to contain Python bytecode.

Correct, Hatchling hard-codes that exclusion.

Reply:
Created https://github.com/pypa/flit/issues/568

Reply:
That said, even if that is merged, it would imply requiring a very new version of flit. Maybe it makes sense just go with this and have an issue about cleaning it up later.

Reply:
There hasn't been a flit-core release with recursive globs FYI.